docs/internal/DistributedArchitectureGuide.md

```diff
@@ -254,9 +261,104 @@ works in parallel with the storage engine.)
 
 # Autoscaling
 
-(Reactive and proactive autoscaling. Explain that we surface recommendations, how control plane uses it.)
+Autoscaling in ES (Elasticsearch) is not an active process that moves data on its own initiative, but
+rather a recommendation system using cluster and node level statistics. ES Autoscaling is paired with
+an ES Cloud service that periodically polls the ES elected master node for suggested cluster changes.
+The cloud service will add more servers/nodes, or increase the resources of existing nodes.
+
+Autoscaling recommendations are tailored for the user [based on user defined policies][], composed of data
+roles (hot, frozen, etc) and [deciders][]. There's a public [webinar on autoscaling][], as well as the
+public [Autoscaling APIs] docs.
+
+Autoscaling's current implementation is based on storage capacity. It does not yet support scaling related
+to search load. Paired with ES Cloud, autoscaling only scales upward, not downward, except for ML nodes
+that do get scaled up _and_ down.
+
+The external autoscaler, external to ES, can be found in the [elasticsearch-autoscaler][] repository.
+
+[based on user defined policies]: https://www.elastic.co/guide/en/elasticsearch/reference/current/xpack-autoscaling.html
+[deciders]: https://www.elastic.co/guide/en/elasticsearch/reference/current/autoscaling-deciders.html
+[webinar on autoscaling]: https://www.elastic.co/webinars/autoscaling-from-zero-to-production-seamlessly
+[Autoscaling APIs]: https://www.elastic.co/guide/en/elasticsearch/reference/current/autoscaling-apis.html
+[elasticsearch-autoscaler]: https://github.com/elastic/elasticsearch-autoscaler
```

Comment:
This is the autoscaler for serverless, which uses a different set of APIs and concepts.

Reply:
Removed. Thanks for taking a look and catching that